### DISCUSSION TITLE
Weewx.ini multiple topics

### DISCUSSION THREAD

**Discussion by 18Scoobz** [daq-tools/kotori#55] [Q&A]
How do you pull multple topics with the weewx add-on.

**Comment by amotl** [comment]
Hi @18Scoobz,

did you have success submitting measurement data from two or more instances of WeeWX to Kotori? I added more details about your question at https://github.com/daq-tools/kotori/issues/53#issuecomment-868464882. Let me know if you need further assistance.

With kind regards,
Andreas.
